src/components/avatar/README.md

```diff
@@ -0,0 +1,28 @@
+# Avatar
+
+#### Description
+
+A component for rendering profile images
+
+## Usage
+
+```js
+import { Avatar } from '@commercetools-frontend/ui-kit';
+
+<Avatar
+  gravatarHash="20c9c1b252b46ab49d6f7a4cee9c3e68"
+  firstName="John"
+  lastName="Doe"
+  size="s"
+/>;
+```
+
+#### Properties
+
+| Props           | Type     | Required | Values                | Default | Description                                      |
+| --------------- | -------- | :------: | --------------------- | ------- | ------------------------------------------------ |
+| `gravatarHash`  | `string` |    ✅    | -                     | -       | Hash of the image to be showed                   |
+| `firstName`     | `string` |          | -                     | ''      | First name of the user in case there is no image |
+| `lastName`      | `string` |          | -                     | ''      | Last name of the user in case there is no image  |
+| `size`          | `string` |    ✅    | oneOf(['s', 'm', 'l]) | `s`     | The size of the Avatar component.                |
+| `isHighlighted` | `bool`   |    -     | -                     | `false` | Highlights the Avatar                            |
```

Comment:
@qmateub Can you explain what "highlights" means? Is it a different look for a mouseover state? When would we use `isHighlighted`?

Reply:
Seems like it is indeed used for the hover effect. In that case I would rename it to `isHovered`. I guess the use-case for it is that we want to show the hovered version when something like a user card is hovered. So the element itself isn't hovered directly but it still needs to look as-if it was hovered?

We can also document that no hover styles will be applied unless this property is passed in.